refactor: update repo and code structure for controller/dto/model pattern

AirKyzzZ opened the pull request.

lotharking commented on docker-dev/docker-compose.yml:
If you want to avoid loss messages across the service, I recommend using a Postgres instance

AirKyzzZ commented:
**@lotharking** Addressed both comments:

1. **package.json**: Added `"packageManager": "pnpm@10.26.0"` for consistent versioning
2. **docker-compose.yml**: Added Postgres 16 service with persistent volume to prevent message loss. All credentials use environment variables (no hardcoded passwords).

**Note**: A new `POSTGRES_PASSWORD` environment variable is required in your `.env` file for the database connection.

Updated `docker-dev/README.md` with Postgres configuration details. Ready for review.

---

lotharking commented on docker-dev/docker-compose.yml:
```diff
@@ -0,0 +1,69 @@
+      - AGENT_INVITATION_IMAGE_URL=https://your-ngrok-domain.ngrok-free.app/logo.png
+      - AGENT_LABEL=Concieragent
+      - USE_CORS=true
+      - EVENTS_BASE_URL=https://your-ngrok-domain.ngrok-free.app
```
I recommended setting the following environment variables in the VS Agent 
```suggestion
      - EVENTS_BASE_URL=https://your-ngrok-domain.ngrok-free.app
      - POSTGRES_HOST=postgres
      - POSTGRES_USER=${POSTGRES_USER:-postgres}
      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
```

AirKyzzZ commented:
**@lotharking** 
I updated `EVENTS_BASE_URL` on line 17 to use environment variables instead of a hardcoded placeholder. It now uses `${EVENTS_BASE_URL}` with a fallback to the ngrok domain.

Also found  out there were some missing variables (`AGENT_PUBLIC_DID`, `AGENT_INVITATION_IMAGE_URL`) 

Added the wallet environment variables:
- `AGENT_WALLET_ID` , defaults to "Concieragent" if not set
- `AGENT_WALLET_KEY` , defaults to "Concieragent" if not set

All values now use environment variables (no hardcoded values).
